src/NuGet.Core/NuGet.Packaging/Rules/AnalysisResources.resx

```diff
@@ -129,6 +129,12 @@
   <data name="FilePathTooLongWarning" xml:space="preserve">
     <value>The file '{0}' path, name, or both are too long. Your package might not work without long file path support. Please shorten the file path or file name.</value>
   </data>
+  <data name="IconUrlAndIconWarning" xml:space="preserve">
+    <value>Detected 'iconUrl' and 'icon' elements. Learn more at aka.ms/nugeticonurl</value>
```

Comment:
Should `aka.ms/nugeticonurl` exist already (or is there an issue tracking creating this URL)?

Reply:
The tracking issue for the documentation is: https://github.com/NuGet/Home/issues/8240

Reply:
The new url now exists: `https://aka.ms/deprecateIconUrl`